src/gwt/acesupport/acemode/r_highlight_rules.js

```diff
@@ -17,491 +17,473 @@
  * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
  *
  */
-
 var $colorFunctionCalls = false;
 
 define("mode/r_highlight_rules", ["require", "exports", "module"], function(require, exports, module)
 {
-   function include(rules) {
-      var result = new Array(rules.length);
-      for (var i = 0; i < rules.length; i++) {
-         result[i] = {include: rules[i]};
+  function include(rules) {
```

Comment:
Sorry this was IntelliJ just doing its thing, I didn't even see this change. JS files should be changed to 2 spaces instead of 3 but I didn't intend for it here. For this case here you can just click the gear at the top and click "hide whitespace changes" or append `?w=1` to the URL for easier viewing.

Reply:
Ah, I didn't know about that option to hide whitespace changes. Nifty!